.github/workflows/dockerfile-image.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
-    types: [labeled, review_requested]
```

Comment:
@vedpatwardhan I'm not sure about this one.

Reply:
Yeah I think we can remove it from here as well 👍

Reply:
Sure 👍🏻